armi/operators/operator.py

```diff
@@ -402,10 +402,6 @@ def _timeNodeLoop(self, cycle, timeNode):
     def _performTightCoupling(self, cycle: int, timeNode: int, writeDB: bool = True):
         """If requested, perform tight coupling and write out database.
 
-        .. impl:: The operator shall allow tight coupling between physics systems.
-            :id: I_ARMI_OPERATOR_PHYSICS
-            :implements: R_ARMI_OPERATOR_PHYSICS
-
```

Comment:
I assume this was removed because it was on a private method, but did you replace it somewhere?

Reply:
Yup!

It is on the `TightCoupler` class in `armi/interfaces.py`, and it is on `Operator.interactAllCoupled()` in `armi/operators/operator.py`.